release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -1394,6 +1394,14 @@ The `oc adm must-gather` command no longer collects audit logs by default. You m
 $ oc adm must-gather -- /usr/bin/gather_audit_logs
 ----
 
+[discrete]
+[id="ocp-4-6-binary-sha256sum-txt-renamed-for-ocp-releases"]
+==== Binary `sha256sum.txt` file has been renamed for {product-title} releases
```

Comment:
Should this be `sha256sum.txt.sig`, as the first sentence specifies?

Reply:
Yeah, good point. :+1: